docs/home/index.md

```diff
@@ -14,9 +14,14 @@ orphan: true
 
 # Welcome to CrateDB!
 
-CrateDB is designed to handle your demanding analytics and search workloads with
-ease. Here you'll find comprehensive resources including reference manuals,
-tutorials, example code, and more.
+CrateDB is a hyper-fast open source multi-model database combining the
+simplicity of SQL and flexible schemas with the scalability of a distributed
+architecture. CrateDB is a multi-model database. Its strength lies in the
+efficient handling of multiple data models within the same database and even
+within the same record in a table. This allows you to use one database for all
+structured, semi-structured and unstructured data. For example, tabular data, 
+JSON or documents, time series, vectors, geospatial, full text 
+and binary objects
```

Comment:
Can this sentence be improved a bit?

/cc @simonprickett

Reply:
Maybe something like:

> This enables you to manage structured, semi-structured, and unstructured data within a single database. CrateDB supports diverse data types such as tabular data, JSON documents, time-series data, vectors, geospatial information, full-text search, and binary objects.